docs/general/serializers.md

```diff
@@ -74,6 +74,18 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+Polymorphic relationships are serialized by specifying the relationship, like any other association. For example:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable
+end
+```
+
+For more context, see the tests for each adapter: [attributes](https://github.com/rails-api/active_model_serializers/blob/master/test/adapter/attributes/polymorphic_test.rb), [json_api](https://github.com/rails-api/active_model_serializers/blob/master/test/adapter/json_api/polymorphic_test.rb), [json](https://github.com/rails-api/active_model_serializers/blob/master/test/adapter/json/polymorphic_test.rb)
```

Comment:
Or ref a link witha conmit and highlighted line numbers (hint, when viewing a file, hit 'y' and adjust the url to point to this repo)